.vib/argo-workflows/vib-verify.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/argo-workflows"
         },
-        "runtime_parameters": "InBvc3RncmVzcWwiOgogICJwb3N0Z3Jlc3FsUGFzc3dvcmQiOiAiMmI3ZFJncUsiCiJzZXJ2ZXIiOgogICJzZXJ2aWNlIjoKICAgICJwb3J0cyI6CiAgICAgICJodHRwIjogODAKICAgICJ0eXBlIjogIkxvYWRCYWxhbmNlciI=",
+        "runtime_parameters": "cmJhYzoKICBzaW5nbGVOYW1lc3BhY2U6IGZhbHNlCmNyZWF0ZUFnZ3JlZ2F0ZVJvbGVzOiB0cnVlCnNlcnZlcjoKICBlbmFibGVkOiB0cnVlCiAgcmJhYzoKICAgIGNyZWF0ZTogdHJ1ZQogIGF1dGg6CiAgICBlbmFibGVkOiB0cnVlCiAgICBtb2RlOiBzZXJ2ZXIKICAgIHNzbzoKICAgICAgZW5hYmxlZDogZmFsc2UKICBjbHVzdGVyV29ya2Zsb3dUZW1wbGF0ZXM6CiAgICBlbmFibGVkOiB0cnVlCiAgICBlbmFibGVFZGl0aW5nOiB0cnVlCiAgc2VjdXJlOiBmYWxzZQogIHNlcnZpY2VBY2NvdW50OgogICAgY3JlYXRlOiB0cnVlCiAgICBhdXRvbW91bnRTZXJ2aWNlQWNjb3VudFRva2VuOiB0cnVlCiAgc2VydmljZToKICAgIHBvcnRzOgogICAgICBodHRwOiA4MAogICAgdHlwZTogTG9hZEJhbGFuY2VyCndvcmtmbG93czoKICByYmFjOgogICAgY3JlYXRlOiB0cnVlCnBvc3RncmVzcWw6CiAgZW5hYmxlZDogdHJ1ZQogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgcG9zdGdyZXNxbDogNTQzMgpteXNxbDoKICBlbmFibGVkOiBmYWxzZQ==",
```

Comment:
I believe the following properties are not relevant:

- `server.sso` (the mode is set to `server`)
- `server.serviceAccount` (the container is scratch and can't be checked)

Reply:
You are right, thanks for the catch!